cmd/noms/diff/diff_test.go

```diff
@@ -70,7 +72,32 @@ func createStruct(name string, kv ...interface{}) types.Struct {
 	return types.NewStruct(name, fields)
 }
 
-func TestNomsMapdiff(t *testing.T) {
+func pathsFromDiff(v1, v2 types.Value, leftRight bool) []string {
+	dChan := make(chan Difference, 64)
```

Comment:
Please see comment about buffers. I believe buffered channels should be the exception not the rule. The algorithm is only going to be as fast as the diff is produced or consumed anyway, and adding a buffer isn't going to change that, it's just going to use up more memory and make creating the buffer slow.

It helps to respond to comments with "done" or something else if you want to discuss something further. Otherwise I need to go through the PR every time to find what hasn't changed.